Generals/Code/GameEngine/Source/GameClient/MessageStream/CommandXlat.cpp

```diff
@@ -2991,20 +2991,17 @@ GameMessageDisposition CommandTranslator::translateGameMessage(const GameMessage
 			}
 			else
 			{
-				if (!(TheRecorder && TheRecorder->getMode() == RECORDERMODETYPE_PLAYBACK))
+				Bool hide = false;
+				if (TheWindowManager)
 				{
-					Bool hide = false;
```

Comment:
It looks like this window code here is useless and can be removed. The only thing that matters is ToggleControlBar().

Reply:
ok